spec/README.md

```diff
@@ -0,0 +1,100 @@
+- [Network Remote ID](https://www.astm.org/f3411-22a.html): Mentioned in the AMC/GM document as a acceptable way to comply to Net-RID. This standard specifies interfaces to ensure that 
+- [EuroCAE ED-269](https://eurocae.net/news/posts/2020/june/ed-269-minimum-operational-performance-standard-for-uas-geo-fencing/): Mentioned in the AMC/GM document as a acceptable way to share with Geozone information.
+- [USSP Interoperability](https://www.astm.org/f3548-21.html): Sharing operational intents / flight authorization can be shared via this standard, the `interuss/dss` implements this standard. 
+
+### Services with no / lack of standards
+- Traffic information service: A service that shares air-traffic information via an API, normally this is hosted by a Air-Navigation Service provider (ANSP) or the standards allow for a special types of "Supplemental Data Service Providers"
+- Common information service: A server that hosts and shares ED-269 Geozones and other information, currently there is no standard for it and this server will normally be expected to be hosted by an ANSP and / or an authority. 
+
+## Core modules
+
+### Configuration module
+A JSON based config file will need to be developed that allows the operator / entity / person using the GCS to configure details of the UTM Service provider they want to use. These must be configured via a simple user-interface (or text editor) initially . A REST based adapter that can submit data to USSP interfaces on the internet and use JWT Bearer tokens as a way to interact with the APIs.
+
+__Out of scope__:
+- At this moment only REST based APIs and JWTs as a way to authenticate is in scope, the main reason for this that is that these are mentioned in the UTM standards, we are aware of other ways to authenticate / authorize but these can be added later. 
+
+__Deliverables__:
+- JSON specification that contains the details of an USSP implementation that the QGCS user-interface can read. 
+- QGCS module / code that reads the JSON and interacts with the user-interface
+
+### HTTP / network module
+The core behavior of the internals of the adapter will be using HTTP REST based services. A high performance HTTP module that makes requests and processes responses is needed to handle the interaction with the USSP (per the config file)
+
+__Deliverables__:
+- A REST based component that interacts with the USSP APIs
+
+### Notifications
+Interaction with USSP and USSP network is a synchronous process and there needs to be a notification mechanism that tells the operator "prominently" what is going on in the UTM system. There is no standard for providing notifications to the operators but since some of the notifications will be safety critical. We are cognizant that there might be a standard that will be developed and the notifications mechanism might need to be adapted to the standard. 
```

Comment:
BTW: The same topic was raised in WG-105 SG-2 UTM by multiple UTM stakeholders. Coordination is required.

Reply:
Thank you, no action required at this point.